docs/questions/query-builder/expressions/in.md

```diff
@@ -18,13 +18,19 @@ in(value1, value2, ...)
 
 Metabase will return rows where the `value1` equals `value2` OR `value3`, etc. Matches must be exact (e.g., strings are case sensitive).
 
-You can choose multiple columns. For example, let's say you wanted to look for the string `Gadget` in both the `[Title]` and `[Category]` columns. You could write:
+For example,
 
 ```
-in("Gadget", [Title], [Category])
+in([Category], "Gadget", "Widget")
```

Comment:
This is a much more common use case (in fact, the use case that we added this function for), so it's weird that we don't have an example for it.